### DISCUSSION TITLE
Add an option for yt-dlp sponsorblock integration (--sponsorblock-remove)

### DISCUSSION THREAD

**Discussion by svblsvljb** [JunkFood02/Seal#136] [Ideas]
This yt-dlp feature is really important, i think, so it would be very nice if seal as a yt-dlp wrapper had it

**Comment by BobbyESP** [comment]
Did you tried to use custom commands to add it? Im gonna try

**Reply by BobbyESP** [reply]
You know how the command works? If you do, just go to Seal > Settings > Downloads (scrolldown to the end) > Click on Custom command > Command template and just add the arguments that you want.

**Reply by rodrigoswz** [reply]
This custom command works for me, remove all sponsorblock categories:

`
--sponsorblock-remove all --no-mtime -f "bv*[ext=mp4]+ba[ext=m4a]/b[ext=mp4] / bv*+ba/b"
`

But getting into the discussion, I like this idea of ​​having Sponsorblock natively as an option when downloading.

This makes it easier to choose which categories to remove and in which video quality to download.

**Comment by svblsvljb** [comment]
> Did you tried to use custom commands to add it? Im gonna try

It should work in custom commands, but adding an option for it will give you ability to use it with other options

**Reply by JunkFood02** [reply]
I haven't use it before tho. if you're familiar with this API, you can use custom command to help test it out or provide us some use cases for example

**Reply by rodrigoswz** [reply]
> I haven't use it before tho. if you're familiar with this API, you can use custom command to help test it out or provide us some use cases for example

https://github.com/JunkFood02/Seal/discussions/136#discussioncomment-3467070

**Comment by JunkFood02** [comment]
Added in v1.2.0-beta.4
